### PR TITLE
Added entries to CODEOWNERS

### PR DESCRIPTION
**Pull request by rzwiefel**
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by rzwiefel** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

File: CODEOWNERS [hunk 1]
```diff
@@ -43,12 +43,20 @@ distribution/test/          @codice/test
 
 catalog/core/                   @codice/core-apis
 
+catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ExportCommand.java		@rzwiefel
```
Comment (coyotesqrl) [inline CODEOWNERS]: ❗️ Need to add `@codice/core-apis` to these two lines.

File: CODEOWNERS [hunk 2]
```diff
@@ -43,12 +43,20 @@ distribution/test/          @codice/test
...
+catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java		@rzwiefel
+
 catalog/core/catalog-core-contentresourcereader/    @codice/core-apis @stustison
 
 catalog/core/catalog-core-localstorageprovider/	    @codice/core-apis @stustison
 
 catalog/core/catalog-core-standardframework/src/main/**/impl/		  @codice/core-apis @stustison
 catalog/core/catalog-core-standardframework/src/main/**/resource/     @codice/core-apis @stustison
+catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/history/		@rzwiefel
```
Comment (coyotesqrl) [inline CODEOWNERS]: ❗️ Need to add `@codice/core-apis` and `@stustison` to this line.

File: CODEOWNERS [hunk 3]
```diff
@@ -43,12 +43,20 @@ distribution/test/          @codice/test
...
+
+catalog/core/catalog-core-validationparser/ 	@rzwiefel
```
Comment (coyotesqrl) [inline CODEOWNERS]: ❗️ Need to add `@codice/core-apis` and `@stustison` to this line.

File: CODEOWNERS [hunk 4]
```diff
@@ -43,12 +43,20 @@ distribution/test/          @codice/test
...
+
+catalog/core/catalog-core-versioning/*				@rzwiefel
```
Comment (coyotesqrl) [inline CODEOWNERS]: ❗️ Need to add `@codice/core-apis` and `@stustison` to this line.

File: CODEOWNERS [hunk 5]
```diff
@@ -60,7 +68,10 @@ catalog/spatial/csw/            @bdeining
 
 catalog/spatial/registry/       @clockard
 
-catalog/ui/catalog-ui-search/   @andrewkfiedler
+catalog/ui/catalog-ui-search/                   @andrewkfiedler
+catalog/ui/catalog-ui-search/pom.xml            @andrewkfiedler @rzwiefel
+catalog/ui/catalog-ui-search/src/**/java/       @rzwiefel
```
Comment (coyotesqrl) [inline CODEOWNERS]: ✏️ Unless you specifically meant to remove `andrewkfielder` from these two lines, you'll need to add him back.

Comment (rzwiefel) [reply CODEOWNERS]: I discussed with him already and did mean to remove him from the backend code.

**Comment by rzwiefel** [comment]
I think instead of cascading the top level owners to every single sub file (at least in this case for history, intrigue backend, import/export command) I think it makes sense to just leave them as-is. Otherwise A) we're going to be over tagging people B) It makes more sense to have the person closest listed instead of everyone at every level. I would instead prefer to rely on the listed owner delegating to the correct team (eg, core-api's here) than have the team tagged every single time unecessarily. 
Of course the benefit personally for myself is not lessened by having the core-api team listed on all of them, so if there are any strong feelings/reasons for this I would be happy to change it.

**Comment by rzwiefel** [comment]
I talked with @coyotesqrl and he said though he had overarching concerns with the upkeep of the file in general, with this particular instance he is 'fine' with it as is. Since he is on vacation for this week, I am proxying his vote to approved. 😄  

If there are no more issues I will merge later today.

**Pull request merged**